mesheryctl/pkg/utils/helpers.go

```diff
@@ -450,3 +450,26 @@ func ContentTypeIsHTML(resp *http.Response) bool {
 	}
 	return false
 }
+
+func SearchAndReplace(path, oldString, newString string) error {
+	fileData, err := ioutil.ReadFile(path)
+	if err != nil {
+		log.Errorf("unable to read file: %v", err)
+		return err
+	}
+
+	if !strings.Contains(string(fileData), oldString) {
+		log.Errorf("unable to replace with %s", newString)
+		return errors.New("unable to find requested string")
+	}
+
+	outputFile := strings.ReplaceAll(string(fileData), oldString, newString)
+
+	err = ioutil.WriteFile(path, []byte(outputFile), 0644)
+	if err != nil {
+		log.Errorf("unable to write into file: %v", err)
```

Comment:
Perhaps, we could change this to: "Subscription to $releaseChannel" not successful. Could not write to Meshery configuration file: err"

Reply:
I was thinking of handling this in the channel command, as I wanted to be the following function be a general utility not specific to channel command/